chaos-monkey-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -99,6 +99,50 @@ CAUTION: Each Chaos Monkey version is built for a specific Spring Boot version,
 
 It is your decision how you want to integrate and use the CM4SB. There is no difference between the two possibilities. However, it's easier to add it as a dependency (see <<_as_dependency>>), so you don't need to adjust the starting of your application.
 
+=== Preparing your application
+
+If you want chaos monkey to attack your classes they need to be open (=non-final). If you're using Java, this is the default. Just don't mark your classes or methods as final. If you're using Kotlin, we recommend using the https://kotlinlang.org/docs/all-open-plugin.html#spring-support[Kotlin Spring plugin], which automatically opens Spring annotated classes for you:
```

Comment:
I personally probably would remove the sentence "Just don't mark your classes or methods as final." 
Or put it in braces after this is the default (as long as you don't mark them final).


Thoughts for future:
We might should consider moving this in a (currently not existing) troubleshoot section. 

Then this could be shortened and the Kotlin part could be moved to the beginning: "In case you use Kotlin, make sure you have the Kotlin Spring Plugin enabled, which is responsible for opening Spring annotated classes."

And the java part (where it usually shouldnt be a problem) could come at the end.
"In case of Java, just make sure to not manually mark the assaulted class as final".

Reply:
> I personally probably would remove the sentence "Just don't mark your classes or methods as final."
Or put it in braces after "...this is the default (as long as you don't mark them final)". What do you think?

Went for the second one.

> I want to have the docs to read in order that you get ready to do chaos as short as possible

As this is a necessity for kotlin, I don't think it is too misplaced in the setup category. Wouldn't be opposed to a troubleshooting second either.